maintainers/maintainer-list.nix

```diff
@@ -100,6 +100,12 @@
     githubId = 241628;
     name = "Adam Russell";
   };
+  a12l = {
```

Comment:
Please add yourself as a maintainer by adding an alphabetically sorted entry to `maintainers/maintainer-list.nix` and commit this change into a separate commit with the message ``maintainers: add a12l``.

Reply:
Done